### DISCUSSION TITLE
How to get all the following data according to the prefix of the key

### DISCUSSION THREAD

**Discussion by landonzeng** [MichaCo/CacheManager#352] [Q&A]
![QQ截图20210704174451](https://user-images.githubusercontent.com/7336781/124380523-9466ed80-dcef-11eb-8811-02a41e66fa3d.png)
I have a data structure like this in redis and I want to get all the data under HAYY:DeviceEXT based on the prefix of the key, which are all the same entities

**Comment by MichaConrad** [comment]
Has been answered a few times, there will be no search in this library. A simple cache does not need to search for keys.
Use Redis directly if you need it, those operations are very expensive.
